src/main/java/com/epam/esm/entity/ExceptionResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.esm.entity;
```

Comment:
This class should be located in some other package. It's not an entity

Reply:
Moved it to exception package